Runtime/Scripts/Component/Output/OscSender.cs

```diff
@@ -43,6 +62,15 @@ void Setup()
             if(Client == null)
                 Client = new OscClient(m_IpAddress, m_Port);
         }
+
+        /// <summary>
+        /// Reinitializes the client. Use this if the IP or port change and the client needs to restart.
+        /// </summary>
+        public void ReInit()
```

Comment:
let's make it a private method i think - keeps the public API smaller, and the functionality just as accessible

also i think `ReInitialize()` isn't too long of a name, and clearer.  sound good ?

Reply:
yep yep. agreed.